src/app/frontend/resource/workloads/daemonset/list/component.ts

```diff
@@ -16,7 +16,7 @@ import {Component} from '@angular/core';
 
 @Component({
   selector: 'kd-daemon-set-list-state',
-  template: '<kd-daemon-set-list></kd-daemon-set-list>',
+  template: './template.html',
```

Comment:
Did you test anything here? The priority of the issue is high and we need it for the next release, if you will not finish it we have to do it anyways.

<img width="440" alt="Zrzut ekranu 2019-07-18 o 09 55 12" src="https://user-images.githubusercontent.com/2823399/61439661-35611000-a942-11e9-801f-42d2bd2d0a5e.png">

Reply:
sorry! by the end of this day, i will fix everything!